src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs

```diff
@@ -9,15 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypto
     {
-        internal static bool GetRandomBytes(ref byte pbBuffer, int count)
+        internal unsafe static bool GetRandomBytes(byte* pbBuffer, int count)
```

Comment:
@jkotas also this wrapper is not so useful...maybe we can remove...what do you think?

Reply:
I think what you have here is fine. I would leave it to comment on the style @bartonjs. There is a variety of styles used for the crypto interop code and I am not sure which one would be preferred here.

Reply:
- Should the error handling be here like what it is in the Apple one?
- Should the `CryptoNative_GetRandomBytes` below return int? It is what the native implementation returns, and it would be a bit more efficient.